packages/indexer-agent/src/commands/start.ts

```diff
@@ -77,8 +77,9 @@ export default {
       .option('gas-price-max', {
         description: 'The maximum gas price (gwei) to use for transactions',
         type: 'number',
-        default: 2000000000,
+        default: 50,
```

Comment:
In the PR description it says that the default is 20 gwei

Reply:
Oops, ya I bumped it up higher since 20 gwei limit will likely get hit pretty often. 

I've now updated the PR comment to reflect the change.